### PR TITLE
Added Publish multi-Architecture debian packages for Rust

### PR DESCRIPTION
**Pull request by KN4CK3R**
PR for review for #10.

### CODE REVIEW

File: content/rust/package/publish-deb-release-multi-arch.md [hunk 1]
```diff
@@ -0,0 +1,468 @@
+---
+date: 2000-01-01T00:00:00+00:00
+title: Publish multi-Architecture Debian packages for Rust
+author: KN4CK3R
+draft: true
+image: /static/header-images/rust-deb.png
```
Comment (bradrydzewski) [inline content/rust/package/publish-deb-release-multi-arch.md]: this can be removed (and the header-images) because we have an illustrator creating images for each tutorial  :)

File: content/rust/package/publish-deb-release-multi-arch.md [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+Setting up hello-world (1.0-1) ...
+$ hello-world
+Hello, world!
+```
+
+# Simplify Package Creation
+
+The package creation should be automated to better fit into a continuous integration environment.
+A simple shell script (call it `pack.sh`) can combine all needed steps and can be transformed into a Drone plugin later.
+
```
Comment (ashwilliams1) [inline content/rust/package/publish-deb-release-multi-arch.md]: While I appreciate creating a plugin, I think this is outside the scope of this tutorial and we want to keep these tutorials focused. I think creating a custom plugin would be a great tutorial by itself. Is this something you would be interested in?

For this tutorial can we invoke the commands directly in the `commands` section of the pipeline instead of creating a plugin?

Comment (KN4CK3R) [reply content/rust/package/publish-deb-release-multi-arch.md]: Sure, I just followed the DRY principle to not fill the yml with bloat.

File: content/rust/package/publish-deb-release-multi-arch.md (hunk 1 above)
Comment (KN4CK3R) [reply content/rust/package/publish-deb-release-multi-arch.md]: I was hoping that you don't need my paint skills. :wink:

**Comment by bradrydzewski** [comment]
looks great, thanks for making the adjustments! Also, can we use `a****@oldschoolhack.me` (your git email address) to send you the bounties for these articles?

**Comment by KN4CK3R** [comment]
Yes, please use this email.
